benchmarks/generate_data_footer_benchmark.py

```
@@ -0,0 +1,570 @@
+import csv
+from enum import Enum
+import json
+import os
+import random
+import string
+from decimal import Decimal
+
+
+class ColumnType(Enum):
+    INT64 = "FLS_I64"
+    BIGINT = "FLS_I64"  # alias for INT64
```

Ruff (0.11.9)

Enum contains duplicate value: `"FLS_I64"`

(PIE796)